releases/Gemfile

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+gem "aws_sdk"
+gem "bunny"
+gem "concurrent-ruby"
+gem "dalli"
+gem "delayed_job"
+gem "ethon"
+gem "excon"
+gem "faraday"
+gem "grape"
+gem "graphql"
+gem "http"
+gem "httpclient"
+gem "koala"
+gem "lmdb"
+gem "mongo"
+gem "mysql2"
+gem "pg"
+gem "que"
+gem "racecar"
+gem "rack"
+gem "rails"
+gem "rake"
+gem "rdkafka"
+gem "redis"
+gem "resque"
+gem "rest-client"
+gem "rspec"
+gem "ruby-kafka"
+gem "sidekiq"
+gem "sinatra"
+gem "trilogy"
```

Comment:
The gems listed above are the ones we want to instrument, not the instrumentations themselves.

Later in the Gemfile I'm iterating over the gemspecs and declaring all of the instrumentation libraries because I want to _force_ the comparability check against the latest version listed in the repo. This ensures that ’all’ was released with the latest version of an instrumentation. 

E.g. say I released Sinatra but forgot to bump it and release All.

The script will force the latest version of the Sinatra instrumentation but then fail because all has a older version of the gem as a requirement. 

Let me know if that explanation helps. 

Do you see any flaws or causes of concern?

Reply:
Thank you, @arielvalentin for the clarification. I was a bit hasty in my comment. When I read this, I read the true libraries as the `opentelemetry-instrumentation-*` versions of the libraries.